server/engine/src/main/java/org/eclipse/lsp/cobol/implicitDialects/sql/Db2SqlVisitor.java

```diff
@@ -457,7 +457,7 @@ private Pair<ExecSqlWheneverNode.WheneverType, String> getWheneverType(Db2SqlPar
             ParseTree pt1 = ruleContext.getChild(2);
             if (Objects.equals(pt1.getText(), "DO")) {
                 result = Pair.of(ExecSqlWheneverNode.WheneverType.DO, ruleContext.getChild(3).getText());
-            } else if (Objects.equals(pt1.getText(), "GO TO") || Objects.equals(pt1.getText(), "GOTO")) {
+            } else if (Objects.equals(pt1.getText(), "GO") || Objects.equals(pt1.getText(), "GOTO")) {
```

Comment:
Just a doubt -  should it be a case insensitive regex match between `pt1.getText.trim()` with` (?i)GO\s*TO`  ?